packages/dds/merge-tree/package.json

```diff
@@ -11,8 +11,30 @@
 	"license": "MIT",
 	"author": "Microsoft and contributors",
 	"sideEffects": false,
-	"main": "dist/index.js",
-	"module": "lib/index.js",
+	"exports": {
+		".": {
+			"import": {
+				"types": "./lib/index.d.mts",
+				"default": "./lib/index.mjs"
+			},
+			"require": {
+				"types": "./dist/index.d.ts",
+				"default": "./dist/index.cjs"
+			}
+		},
+		"./dist/test": {
```

Comment:
Would it be better to just call this `./test`? Would clean up some import patterns nicely.

Reply:
Sure, but in a future change, because doing that require more churn.